helpers.ts

```diff
@@ -0,0 +1,12 @@
+import {User} from "./interfaces";
+
+const getHighestFromArr = (arr: Array<number>) => {
+    if (arr.length === 0) {
```

Comment:
you can do `if (!arr.length)`

Reply:
actually you can make it even shorter by doing `return arr.lenght ? Math.max(...arr) + 1 : 0;`